content/doc/tutorials/using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline.adoc

```diff
@@ -0,0 +1,308 @@
+---
+layout: documentation
+title: Build a Jenkins pipeline by using Jenkinsfile Runner GitHub Actions
+section: doc
+---
+
+:toc:
+:toclevels: 3
+:imagesdir: ../../book/resources
+
+This tutorial shows you how to use link:https://jenkinsci.github.io/jfr-action-doc/[Jenkinsfile Runner GitHub Actions] in the link:https://github.com/features/actions[GitHub Actions] context. 
+If you want to use Jenkins pipeline in GitHub Actions context, this tutorial can help you.
+
+=== The Concept and How It Works
+
+Classical Jenkins instances bind physically with the permanently running servers. 
```

Comment:
"Physically" means the classical Jenkins instance has close relationship with the physical machine. From my point of view, Jenkinsfile Runner enables decoupling (to some extent) between Jenkins instance and physical machine because the Jenkins instance is ephemeral in this case. So Jenkinsfile Runner GitHub Actions is a simple user case/implementation of Jenkinsfile Runner.

These are my personal thoughts during GSoC. Maybe my understanding is wrong so please correct me.

Reply:
Hi @Cr1t-GYM no worries at all! I was reading it as a more literal "physically", and wanted to be sure. That makes sense to me, so you can disregard this suggestion.